skrub/tests/test_interpolation_join.py

```diff
@@ -27,11 +35,23 @@ def weather():
     )
 
 
+@pytest.mark.parametrize("px", MODULES)
 @pytest.mark.parametrize("key", [["latitude", "longitude"], "latitude"])
 @pytest.mark.parametrize("with_nulls", [False, True])
-def test_interpolation_join(buildings, weather, key, with_nulls):
+def test_interpolation_join(px, buildings, weather, key, with_nulls):
```

Comment:
You have errors because you need to "shield" the call to `pl` with `if POLARS_SETUP`. This could be done with:
```python
if POLARS_SETUP and px is pl:
```
(note that when the first element is False, the second element won't be evaluated)

Reply:
Solved with your above suggestion of calling `pd.DataFrame()` first